This patch is needed because the default shell on Solaris 10 has issues.
The default shell on S11 and S12 is ksh93, which would probably work. If
so, then this patch can be removed. This needs to be investigated further.

--- mozilla-esr31/python/mozbuild/mozbuild/mozconfig_loader.orig	2015-06-04 17:42:52.695582248 -0700
+++ mozilla-esr31/python/mozbuild/mozbuild/mozconfig_loader	2015-06-04 17:42:52.698042560 -0700
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
